src/common/components/DataTable/Filters.tsx

```diff
@@ -107,19 +110,23 @@ export const RecentDateFilter = (days: number[]) => {
             className='mb-1'
             type='radio'
             name={attribute}
-            label='All'
+            label={t('all')}
           />
         </Form.Group>
 
-        {days.map(d => (
-          <Form.Group key={d} controlId={`${attribute}-days${d}`}>
+        {days.map(count => (
+          <Form.Group key={count} controlId={`${attribute}-days${count}`}>
             <Form.Check
-              checked={activeFilters.length > 0 && choice === d}
+              checked={activeFilters.length > 0 && choice === count}
               className='mb-1'
-              onChange={() => filterDaysAgo(d)}
+              onChange={() => filterDaysAgo(count)}
               type='radio'
               name={attribute}
-              label={`In the last ${d} days`}
+              label={
+                <Trans ns='common' i18nKey='lastNDays' count={count}>
+                  In the last {{ count }} dayz
```

Comment:
This text within the component is not actually used, but left here to help provide context when viewing the component. (The text is replaced with what is in the translation files)